src/_guides/language/language-tour.md

```diff
@@ -1358,6 +1347,13 @@ void main() {
 All functions return a value. If no return value is specified, the
 statement `return null;` is implicitly appended to the function body.
 
+<?code-excerpt "test/language_tour/functions_test.dart (implicit-return-null)"?>
+{% prettify dart %}
+foo() {}
+
+assert(foo() == null);
+{% endprettify %}
+
```

Comment:
I added a short code excerpt in support of the prose.

Reply:
Is this in the Return values section?